src/components/ToggleSwitch/index.tsx

```diff
@@ -80,15 +80,15 @@ const styles = StyleSheet.create({
   },
   labelLeft: {
     marginRight: dimen.normalizeDimen(5),
-    color: COLORS.white,
-    fontSize: normalizeFontSize(12),
-    fontWeight: '400'
+    color: COLORS.gray400,
+    fontSize: normalizeFontSize(8),
+    fontFamily: fonts.inter[400]
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color and font size of the `labelLeft` style have been changed. The color has been changed from `COLORS.white` to `COLORS.gray400`, and the font size has been reduced. Additionally, the `fontWeight` property has been replaced with the `fontFamily` property. Make sure these changes do not affect readability and visibility of the label.
```diff
-    color: COLORS.white,
-    fontSize: normalizeFontSize(12),
-    fontWeight: '400'
+    color: COLORS.gray400,
+    fontSize: normalizeFontSize(8),
+    fontFamily: fonts.inter[400]
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->